.github/workflows/test_vulcan.yml

```diff
@@ -30,6 +30,18 @@ jobs:
       - name: Setting up a secure repository
         run: git config --global --add safe.directory /__w/azion/azion
 
-      - name: test vulcan
+      # - name: test vulcan
```

Comment:
Does needed this comment?

Reply:
I can remove it... both you and Max pointed it out!